attachments.scad

```diff
@@ -2167,7 +2162,7 @@ module edge_profile_asym(
 //   d = Diameter of corner mask.
 //   convexity = Max number of times a line could intersect the perimeter of the mask shape.  Default: 10
 // Side Effects:
-//   Tags the children with "remove" (and hence sets $tag) if no tag is already set.
```

Comment:
This side effect description is still correct, and arguably clearer to the end user.